offline_install_scripts/build_pyinstaller.sh

```diff
@@ -18,7 +18,7 @@ cp BATCH_SCORING_EXECUTABLE_README.txt dist/pyinstaller
 
 pip install -U pip 
 pip install -U urllib3[secure]
-pip install -r requirements.txt -r requirements-pyinstaller.txt
+pip install -r requirements-base.txt -r requirements-pyinstaller.txt
```

Comment:
Not sure about this one, but apparently it requires Py3 (see above), so I assume py27 deps are not needed